lib/hooks.ex

```diff
@@ -85,6 +107,7 @@ defmodule Committee.Hooks do
     #!/bin/sh
 
     #{@identifier}
+    # version:#{Committee.__version__()}
```

Comment:
WDYT of using the identifier as marker for the version? To avoid we getting anything that has `version` in it.

`#{@identifier}:#{Committee.__version__()}`

Reply:
Yeah okay I see where you're coming from, I think you're right. I initially was thinking we could add more stuffs like dates or whatsoever while maintaining the same structure:

```
# committee
# version:1.0
# date:02-04-2020
```

But then again I don't think there's a lot of use for that, so I'm going to follow your suggestion. 👍